packages/components/src/Dialog/Dialog.test.tsx

```diff
@@ -519,3 +519,54 @@ test('close state has a listener', async () => {
 
   expect(spy.mock.calls).toMatchInlineSnapshot(`[]`);
 });
+
+test('cancel button closes dialog', async () => {
+  render(
+    <Dialog.Trigger>
+      <Button>Open</Button>
+
+      <Dialog closeButton>
+        <Dialog.Title>Headline</Dialog.Title>
+        <Dialog.Actions>
+          <Dialog.Cancel />
+        </Dialog.Actions>
+      </Dialog>
+    </Dialog.Trigger>
+  );
+
+  const button = screen.getByText('Open');
+  await user.click(button);
+
+  const cancel = screen.getByText('Cancel');
+  await waitFor(() => {
+    expect(cancel).toBeInTheDocument();
+  });
+
+  await user.click(cancel);
+  await waitFor(() => {
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
+  });
+});
+
+test('render custom cancel button', async () => {
```

Comment:
Is it allowed to have a custom cancel button still? Like before? (anyways the test text must be adjusted)

Reply:
What should be adjusted?

Reply:
What should be adjusted?

Reply:
the test text ('render custom cancel button) because its not custom

Reply:
It's a custom label. Very nitpicky 😜